settings.gradle

```diff
@@ -13,6 +13,7 @@ include ':HLSPlayerID3SampleApp'
 include ':HLSPlayerSampleApp'
 include ':WebVTTSampleApp'
 include ':DFXPClosedCaptioningSampleApp'
+include ':ExoPlayerSampleApp'
```

Comment:
The android-exoplayer-player-${version}.aar artifacts aren't being published to our public maven repo yet, so can you remove this line, so building the samples won't break for customers, please?  Thanks.